e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -156,7 +155,7 @@ describeEE("formatting > whitelabel", () => {
           cy.findByRole("searchbox", {
             name: "Login and unsubscribe pages",
           }).click();
-          mantinePopover().findByText("Custom").click();
+          popover().findByText("Custom").click();
```

Comment:
Reuse the current util. `mantinePopover` was introduce way before `popover` was made to work with Mantine popover.